magneto_lib/magneto.h

```diff
@@ -1,30 +1,33 @@
 #ifndef MAGNETO_LIB_MAGNETO_H
 #define MAGNETO_LIB_MAGNETO_H
-#include <map>
+
 namespace cr {
 namespace magneto {
 
- enum  TypeOfPiece {
-        king,          
-        queen,
-        rook,
-        pawn,       
-        bishop,    
-        knight,
+    enum class TypeOfPiece {
+       kKing,          
+       kQueen,
+       kRook,
+       kPawn,       
+       kBishop,    
+       kKnight,
          
     };
 
-    enum Color{
-        white,
-        black,
+    enum class Color{
+        kWhite,
+        kBlack,
     };
 
     struct Square{
-        int row;
-        int column;
-        std::map <int, std::map<int,std::string>> map;
+private:
+        int file;
+        int rank;
+public:
+	Square(int, int);
```

Comment:
Although not necessary, it's good to have the names of the variables in the construction (or any other function) declaration to make it self explanatory.